src/class/audio/audio_device.c

```diff
@@ -337,6 +336,7 @@ typedef struct
     }compute;
 
   } feedback;
+  CFG_TUSB_MEM_ALIGN uint32_t feedback_value; // Feedback value for asynchronous mode (in 16.16 format).
```

Comment:
we can put the CFG_TUSB_MEM_ALIGN to the struct feedback, which will make the value as 1st member to be aligned correctly ?

Reply:
Yes that will work. I was thinking to avoid aligment on struct member so I moved `value` outside, but it should be ok.